controllers/machinedeletionremediation_controller.go

```diff
@@ -115,8 +115,12 @@ func (r *MachineDeletionRemediationReconciler) deleteMachineOfNode(ctx context.C
 	}
 
 	//verify machine is deleted
-	if err := r.Get(context.TODO(), key, machine); !errors.IsNotFound(err) {
-		r.Log.Info("machine associated to node was not deleted probably due to a finalizer on the machine, note that the remediation is pending", "node name", nodeName)
+	if err := r.Get(context.TODO(), key, machine); !apiErrors.IsNotFound(err) {
```

Comment:
I think this should be changed. IMO it's wrong grouping err == nil & err != nil under the same use case.
For instance the next line `machinePhase, err := getMachineStatusPhase(machine)` isn't relevant for err != nil